greenheart/tools/eco/finance.py

```diff
@@ -1461,6 +1465,21 @@ def run_profast_full_plant_model(
         cost=opex_breakdown["electrolyzer"],
         escalation=gen_inflation,
     )
+    if isinstance(electrolyzer_cost_info["electrolyzer_var_om"], (list, np.ndarray)):
+        vopex_elec = dict(zip(years_of_operation, electrolyzer_cost_info["electrolyzer_var_om"]))
+    elif isinstance(electrolyzer_cost_info["electrolyzer_var_om"], float) and (
```

Comment:
This check feels out of place here. I think the checkout should be done in the electrolyzer costing and set to zero there if needed.